metals/src/main/scala/scala/meta/internal/builds/SbtBuildTool.scala

```diff
@@ -51,31 +51,56 @@ case class SbtBuildTool(version: String) extends BuildTool {
           sbtArgs
         ).flatten
     }
-    writeSbtMetalsPlugin(config)
+    removeOldPlugin()
```

Comment:
Maybe something more telling like:

```suggestion
    removeLegacyGlobalPlugin()
```

(PS: I haven't followed closely, but what's the technical reason for switching to a local plugin?)

Reply:
It doesn't add the plugin for every workspace, which is useful to reduce some amount of overhead and is not encouraged by the sbt docs. Additionally, I don't think we would be able to work with sbt support, because global plugins don't add plugins to meta builds.

Reply:
Fixed the name :)